update to support STAC Catalogs and link objects (#1) (#9)
stac-extensions/file: CHANGELOG.md

### Added

> - Support for STAC Catalogs and link objects. [#9](https://github.com/stac-extensions/file/pull/9)

Reviewer: We can move this to v2.1.0 as it has not been released yet.

Author: Done/pushed.

> - updates to support STAC Catalogs and link objects. [[PR#9]](https://github.com/stac-extensions/file/pull/9#)

Reviewer: ```suggestion
- updates to support STAC Catalogs and link objects. [#9](https://github.com/stac-extensions/file/pull/9)
```

### Added

- `file:local_path` to indicate a relative local path for the asset file. [[PR#7]](https://github.com/stac-extensions/file/pull/7#)

> - updates to support STAC Catalogs and link objects. [#9](https://github.com/stac-extensions/file/pull/9)

Reviewer: ```suggestion
- Support for STAC Catalogs and link objects. [#9](https://github.com/stac-extensions/file/pull/9)
```
